docs/Custom_Device_Types.md

```diff
@@ -0,0 +1,198 @@
+2.	Read asynchronous custom device FIFOs (N-1)
+3.	Run Read Data From HW case of Inline Hardware custom devices
+   •	Scaling is applied after all hardware inputs have been acquired
+4.	Read models from **[Controller](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/cp_controller/)** » **[Simulation Models](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/simulation_models_se/)**
+5.	Read from DPL (N-1)
+6.	Process system mappings1
+7.	Run the Execute Model case of Inline Model custom devices
+   •	All hardware inputs have been acquired and all channels have been scaled
+before this case runs
+8.	Process system mappings1
+9.	Execute generators
+10.	Process system mappings1
+11.	Write to DPL
+12.	Write to **[Controller](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/cp_controller/)** » **[Simulation Models](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/simulation_models_se/)**
+13.	Write hardware outputs to **[Controller](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/cp_controller/)** » **[Hardware](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/hardware_se/)** » **[Chassis](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/chassis_se/)**
+14.	Run the Write to Hardware case of Inline Hardware custom devices
+15.	Write to Asynchronous device FIFOs
+
+You can’t read data from a previous step until a “process system mappings” step has executed, even if that step acquired the data you want. For example, you write an inline HW custom device, and inside the read data from HW state of this custom device, you want to read the channel data from a DAQ card in the configuration. The DAQ executes at step 1, your code executes at step 3. However, if you read the channel for the DAQ in your code in step 3, you would get the data from the previous iteration (N-1) because “process system mappings” hasn’t executed yet. This is the case for NIVS 2010, it will likely change in the future.
+ 
+### Low-Latency Mode
+
+Low latency mode executes models in-line.
+1.	Get hardware inputs from **[Controller](https://zone.ni.com/reference/en-XX/help/372846M-01/veristand/cp_controller/)** » **[Hardware](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/hardware_se/)** » **[Chassis](https://zone.ni.com/reference/en-XX/help/372846B-01/veristand/chassis_se/)**
+•	DAQ Digital Lines and Counters are read after Read From HW case of Inline Hardware custom devices
+2.	Read asynchronous custom device FIFOs (N-1)
+3.	Run the Read Data From HW case of Inline Hardware custom devices
+•	Scaling is applied after all hardware inputs have been acquired
+4.	Read from DPL (N-1)
+5.	Process system mappings1
+6.	Run the Execute Model case of Inline Model custom devices
+•	All hardware inputs have been acquired and all channels have been scaled
+before this case runs
+7.	Process system mappings1
```

Comment:
```suggestion
7.	Executes steps of running real-time sequences.
&nbsp;&nbsp;&nbsp; **Note:** VeriStand executes real-time sequences after input operations but before output operations and continues to run every step of the real-time sequence until the sequence is complete, reaches a Yield step, or completes an iteration of a loop with Auto Yield set to TRUE. If a sequence takes longer than the given time for an iteration of the PCL, the PCL runs late. To avoid errors, break up the timing of the steps by placing Yield steps throughout the sequence and enabling the Auto Yield property for any loops in the sequence.
```